api/BUILD

```diff
@@ -117,19 +88,64 @@ proto_library(
         "//envoy/config/resource_monitor/fixed_heap/v2alpha:pkg",
         "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
         "//envoy/config/retry/previous_priorities:pkg",
+        "//envoy/config/route/v3alpha:pkg",
         "//envoy/config/trace/v2:pkg",
         "//envoy/config/trace/v2alpha:pkg",
         "//envoy/config/trace/v3alpha:pkg",
         "//envoy/config/transport_socket/alts/v2alpha:pkg",
         "//envoy/config/transport_socket/tap/v2alpha:pkg",
-        "//envoy/config/transport_socket/tap/v3alpha:pkg",
         "//envoy/data/accesslog/v2:pkg",
         "//envoy/data/accesslog/v3alpha:pkg",
         "//envoy/data/cluster/v2alpha:pkg",
         "//envoy/data/core/v2alpha:pkg",
         "//envoy/data/core/v3alpha:pkg",
         "//envoy/data/tap/v2alpha:pkg",
         "//envoy/data/tap/v3alpha:pkg",
+        "//envoy/extensions/access_loggers/grpc/v3alpha:pkg",
+        "//envoy/extensions/clusters/aggregate/v3alpha:pkg",
+        "//envoy/extensions/clusters/dynamic_forward_proxy/v3alpha:pkg",
+        "//envoy/extensions/common/dynamic_forward_proxy/v3alpha:pkg",
+        "//envoy/extensions/common/ratelimit/v3alpha:pkg",
+        "//envoy/extensions/common/tap/v3alpha:pkg",
+        "//envoy/extensions/filters/common/fault/v3alpha:pkg",
+        "//envoy/extensions/filters/http/adaptive_concurrency/v3alpha:pkg",
+        "//envoy/extensions/filters/http/buffer/v3alpha:pkg",
+        "//envoy/extensions/filters/http/csrf/v3alpha:pkg",
+        "//envoy/extensions/filters/http/dynamic_forward_proxy/v3alpha:pkg",
+        "//envoy/extensions/filters/http/ext_authz/v3alpha:pkg",
+        "//envoy/extensions/filters/http/fault/v3alpha:pkg",
+        "//envoy/extensions/filters/http/grpc_http1_reverse_bridge/v3alpha:pkg",
+        "//envoy/extensions/filters/http/grpc_json_transcoder/v3alpha:pkg",
+        "//envoy/extensions/filters/http/grpc_stats/v3alpha:pkg",
+        "//envoy/extensions/filters/http/gzip/v3alpha:pkg",
+        "//envoy/extensions/filters/http/header_to_metadata/v3alpha:pkg",
+        "//envoy/extensions/filters/http/health_check/v3alpha:pkg",
+        "//envoy/extensions/filters/http/ip_tagging/v3alpha:pkg",
+        "//envoy/extensions/filters/http/jwt_authn/v3alpha:pkg",
+        "//envoy/extensions/filters/http/lua/v3alpha:pkg",
+        "//envoy/extensions/filters/http/original_src/v3alpha:pkg",
```

Comment:
I like that we are now reflecting a consistent structure with the source tree. One question I have is whether this should all be nested under `envoy.confg.extensions`, but I think the flatter namespace is a bit cleaner for the common case.

Reply:
I considered `envoy.config.extensions`, but went to this because flatter is cleaner, and in the flatter namespace also we don't need separate `envoy.service.extensions` (e.g. RLS, ALS, ext_authz).